pandas/io/stata.py

```diff
@@ -668,7 +668,9 @@ class StataValueLabel:
         Encoding to use for value labels.
     """
 
-    def __init__(self, catarray: Series, encoding: str = "latin-1") -> None:
+    def __init__(
+        self, catarray: Series, encoding: Literal["latin-1", "utf-8"] = "latin-1"
```

Comment:
Not too familiar with stata, but are utf8 and latin1 the only supported/available encodings?

Reply:
Yes, it seems so (based on the discussion here https://github.com/pandas-dev/pandas/pull/21246).